### PR TITLE
added basic visualization automation

### LINKED ISSUE #6: EDA - 1 - Auto-Graph Graph generation module
- [x] Define a structure for the Graph module
- [x] Column-wise distribution graphs for numerical columns
- [x] heat-map for missing values
- [x] Target class pie chart

### PR DESCRIPTION
**Pull request by swainshashwat**
fixes #6

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "D:\\anaconda\\envs\\train\\python.exe"
+}
```
Comment (sandipbhuyan) [inline .vscode/settings.json]: add one extra line here

File: dataprocessing/eda/test.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import pandas as pd
+
+# sample data
+df_tmp = pd.read_csv('sample_data/titanic/train.csv')
+
+v1 = viz.vizualize(df_tmp)
+
+v1.dist_plot(col_tp='_all')
+v1.null_heat_plot()
+v1.class_pie_plot('Survived')
```
Comment (sandipbhuyan) [inline dataprocessing/eda/test.py]: add one extra line where you see   this

File: dataprocessing/eda/viz.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        
+        labels = labels = self.data[class_col].unique()
+        sizes = np.around((np.array(class_counts)/(np.array(class_counts).sum()))*100, decimals=2)
+
+        fig, ax = plt.subplots()
+        ax.pie(sizes,  labels=labels, autopct='%1.1f%%',
+                shadow=True, startangle=90)
+        ax.axis('equal')  # Equal aspect ratio ensures that pie is drawn as a circle.
+        
+        plt.show()
```
Comment (sandipbhuyan) [inline dataprocessing/eda/viz.py]: same here

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+numpy
+pandas
+matplotlib
+sklearn
```
Comment (sandipbhuyan) [inline requirements.txt]: Create one file `CONTRIBUTION.md` and write the setup procedure for it

**Review by sandipbhuyan — COMMENTED** [review]
Do the mentioned changes

**Comment by swainshashwat** [comment]
Done the necessary changes!

File: requirements.txt (hunk 4 above)
Comment (swainshashwat) [reply requirements.txt]: okay

File: .vscode/settings.json (hunk 1 above)
Comment (sandipbhuyan) [inline .vscode/settings.json]: delete this file
`rm -rf .vscode`

**Review by sandipbhuyan — COMMENTED** [review]
All look good
Thanks

**Review by sandipbhuyan — APPROVED** [review]

**Pull request merged**